feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -460,6 +463,217 @@ func configACLInterface(t *testing.T, iFace *oc.Acl_Interface, ifName string) *a
+					},
+					"destination-ip": map[string]any{
+						"prefix": aclNullPrefix,
+					},
+				},
+			},
+		}
+		entry30Update, err := json.Marshal(aclEntry30Val)
+		if err != nil {
+			t.Fatalf("Error with json Marshal: %v", err)
+		}
+
+		gpbSetRequest := &gpb.SetRequest{
+			Prefix: &gpb.Path{
+				Origin: "srl",
+			},
+			Update: []*gpb.Update{
+				{
+					Path: &gpb.Path{
+						Elem: []*gpb.PathElem{
+							{Name: "acl"},
+							{Name: "ipv4-filter", Key: map[string]string{"name": name}},
+							{Name: "entry", Key: map[string]string{"sequence-id": "10"}},
+						},
+					},
+					Val: &gpb.TypedValue{
+						Value: &gpb.TypedValue_JsonIetfVal{
+							JsonIetfVal: entry10Update,
+						},
+					},
+				},
+				{
+					Path: &gpb.Path{
+						Elem: []*gpb.PathElem{
+							{Name: "acl"},
+							{Name: "ipv4-filter", Key: map[string]string{"name": name}},
+							{Name: "entry", Key: map[string]string{"sequence-id": "20"}},
+						},
+					},
+					Val: &gpb.TypedValue{
+						Value: &gpb.TypedValue_JsonIetfVal{
+							JsonIetfVal: entry20Update,
+						},
+					},
+				},
+				{
+					Path: &gpb.Path{
+						Elem: []*gpb.PathElem{
+							{Name: "acl"},
+							{Name: "ipv4-filter", Key: map[string]string{"name": name}},
+							{Name: "entry", Key: map[string]string{"sequence-id": "30"}},
+						},
+					},
+					Val: &gpb.TypedValue{
+						Value: &gpb.TypedValue_JsonIetfVal{
+							JsonIetfVal: entry30Update,
+						},
+					},
+				},
+			},
+		}
+		gnmiClient := d.RawAPIs().GNMI().Default(t)
+		if _, err := gnmiClient.Set(context.Background(), gpbSetRequest); err != nil {
+			t.Fatalf("Unexpected error configuring SRL ACL: %v", err)
+		}
+	default:
+		t.Fatalf("Unsupported vendor, got vendor: %s, want: NOKIA", d.Vendor())
+	}
+}
+
+// Helper function to replicate AdminAllACL() configs in native model
+// Remove ACL entries 10, 20
+// Then craft a gNMI set Request to update the changes.
+func configAdmitAllACLNative(t testing.TB, d *ondatra.DUTDevice, name string) {
+	t.Helper()
+	switch d.Vendor() {
+	case ondatra.NOKIA:
+		gpbDelRequest := &gpb.SetRequest{
+			Prefix: &gpb.Path{
+				Origin: "srl",
+			},
+			Delete: []*gpb.Path{
+				{
+					Elem: []*gpb.PathElem{
+						{Name: "acl"},
+						{Name: "ipv4-filter", Key: map[string]string{"name": name}},
+						{Name: "entry", Key: map[string]string{"sequence-id": "10"}},
+					},
+				},
+				{
+					Elem: []*gpb.PathElem{
+						{Name: "acl"},
+						{Name: "ipv4-filter", Key: map[string]string{"name": name}},
+						{Name: "entry", Key: map[string]string{"sequence-id": "20"}},
+					},
+				},
+			},
+		}
+		gnmiClient := d.RawAPIs().GNMI().Default(t)
+		if _, err := gnmiClient.Set(context.Background(), gpbDelRequest); err != nil {
+			t.Fatalf("Unexpected error removing SRL ACL: %v", err)
+		}
+	default:
+		t.Fatalf("Unsupported vendor, got vendor: %s, want: NOKIA", d.Vendor())
+	}
+}
+
+// Helper function to replicate configACLInterface in native model
```

Comment:
Seems like comment synax is missing comma and period? Note that Golang comment has no length requirement/restriction.